cdk/api/index.ts

```diff
@@ -103,6 +103,18 @@ export default class Api extends Construct {
     });
     elasticsearch.grantReadWrite(searchByKeywords);
 
+    const sparqlFunction = new Function(this, 'SparqlFunction', {
+      functionName: `${stage}-obp-cdk-api-sparql`,
+      handler: 'sparql.handler',
+      runtime: Runtime.NODEJS_14_X,
+      code: Code.fromAsset(path.join(lambdasPath, 'sparql')),
+      description: 'Perform a SPARQL query',
+      timeout: Duration.minutes(5),
+      environment: {
+        SPARQL_URL: `https://${virtuosoHostname}:${8890}/sparql`,
```

Comment:
This is valid for now but will need to be updated after your Neptune work gets implemented. I realize they're different branches though but wanted to make it gets called out.

Reply:
👍